### ISSUE TITLE
Broken link to fsprojects logo

### ISSUE THREAD

**Issue by vasily-kirichenko** [fsprojects/AzureDocumentDbTypeProvider#22]
![image](https://cloud.githubusercontent.com/assets/873919/26006029/a77add6a-3743-11e7-9417-bc961f965a8e.png)

**Comment by dsyme** [comment]
Yes, the repo just got transferred here :)  @stewart-r Are you good to republish the docs with the updated repo location?
